src/main/java/com/autotune/analyzer/exceptions/InvalidConversionOfRecommendationEntryException.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat, IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+
+package com.autotune.analyzer.exceptions;
+
+public class InvalidConversionOfRecommendationEntryException extends Throwable{
```

Comment:
add this exception for ExperimentEntry and resultentry .. may be in separate PR

Reply:
Sure will raise a separate PR